File: scales/image-resize/imageResize.cpp
// -----------------------------------------------------------------------------
//
// Example for image resizing in C++ with OpenCV.
//
// -----------------------------------------------------------------------------

#include <opencv2/opencv.hpp>

int main()
{
    cv::Mat image = cv::imread("image.png");

    image.resize(image.rows / 2);

    cv::imwrite("imageResized.png", image);
}

File: scales/image-resize/imageResize.py
# -----------------------------------------------------------------------------
#
# Example for image resizing in Python3 with OpenCV.
#
# -----------------------------------------------------------------------------

import cv2

image = cv2.imread("image.png");

image.resize(image.shape[0] // 2, image.shape[1], 3);

cv2.imwrite("imageResized.png", image);
